packages/manager/modules/pci/package.json

```diff
@@ -40,6 +40,7 @@
     "@ovh-ux/manager-filters": "^1.1.1",
     "@ovh-ux/manager-models": "^1.0.0",
     "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
+    "@ovh-ux/manager-pci-block-storage": "^0.0.0",
```

Comment:
not needed as a micro app is independent of other apps

Reply:
https://github.com/ovh/manager/pull/10563#discussion_r1472797974